### DISCUSSION TITLE
運動會一定要參加嗎

### DISCUSSION THREAD

**Discussion by Tchang323** [1101-GS4719-C/Python-Community-GS4719-C#2251] [General]
運動會這個一年一度的盛大活動，對於大一新生來說，好像有規定一定要參加？聽學長姐說，這也是一個很好的機會可以藉此和同學們變得更熟悉。不過我還在猶豫要不要參加，因為覺得有點浪費時間哈哈

**Comment by Chattenoire** [comment]
沒時間就不要參加吧，我們那班當初運動會走個100公尺進場完，所有人都集體落跑了。
至於校外時數，如果有缺可以去當志工；或者從其他地方拿，像是狗狗貓貓社團

**Comment by gigitilto** [comment]
我覺得可以參加，但對不同的人來說，也不一定需要參加，參加的理由是如果要做運動會志工拿校外時數，我覺得就很好，但假如沒有急著要校外時數，亦或是有其他事情要做的話，例如我，大一大二的運動會我就都是全程沒有參與的，窩在實驗室做實驗。

**Comment by christaaah** [comment]
作為參加過兩年的人來說，我覺得可以趁大一的時候參加，而且我們系還有額外的獎勵，參加一次就可以得到很多全家禮券我覺得還蠻划的，但是如果覺得忙得話那不參加也沒關係。

**Comment by Boying12** [comment]
非大一的話，對運動有興趣的話報名幾個項目玩玩其實是不錯的體驗，不會花太多時間，當然，沒興趣的話就不用參加。至於大一的話，個人是有參加幾個項目，認識一些系上學長跟白瞟餐盒也還行。

**Comment by qiu0109** [comment]
對於有在爭大老鷹的學系，我覺得運動會一定要參加，就算不會運動也可以報名趣味競賽，像我大一時就趣味競賽幾乎都有上場，我覺得還蠻好玩的。參加運動會可以增加系上的向心力，然後認識很多學長姐，是還蠻不錯的活動。但如果系上本身對運動會沒有熱情的話，不如早早回家睡覺，待著也只是浪費時間而已。

**Comment by wcyeee** [comment]
我覺得如果有時間的話就可以去，沒時間或是單純想要逼自己去認識人的話可以休息就好！但如果是很熱情的系的話，感覺也可以去現場感受一下系上的熱情也許被感染之後下場玩就會很有趣了！

**Comment by Mminniehuang** [comment]
雖然運動會已經結束了，但是還是想來分享一下感想。
去年大一的時候我沒有參加運動會，今年為了校外時數所以選擇留在學校。其實感覺相較於國高中，大家對於參與運動會的熱情真的少很多，也比較感受不到那種團隊凝聚的力量。（尤其第二天還下雨）
比較有趣的是，我覺得最積極參與的反而是是教職員工，各項目報名和參加的組數都是最多的🤣

**Comment by yuan3x2** [comment]
雖然運動會已經過了，但或許可以當作你明年的參考，我自己是幾乎每年運動會都有參加，雖然有一種浪費時間的感覺，但說實話當你融入了運動會的氛圍後會發現其實蠻有趣的，有一種我還年輕的感覺，再加上畢業之後應該就很少會有這種機會了，因此我認為不排斥參與活動的話不妨參加看看～

**Comment by ed112601513** [comment]
我去年運動會繞了一次場後就不太想參加了，覺得沒什麼意義又浪費時間，還不如在房間準備考試和睡覺，但這次因為有課程說去繞場就加分所以我還是不情願地去繞，去到那邊才發現除了修那門課的學生外，其他來繞場的系上學生一隻手數的出來，最主要的原因我覺得還是因為沒有誘因，所以沒參加比賽的基本上都不會去。

**Comment by lai1008** [comment]
我覺得相較於國高中，大家對於參與運動會的熱情真的少很多，也比較感受不到那種團隊凝聚的力量，而且第二天甚至還下雨就更掃興了。但參加的人數少拿獎牌的機率也會比較高。我覺得最有趣的是教職員工是最積極參與比賽的，各項目報名和參加的組數都超多

**Comment by Elaine4488** [comment]
我大一的時候進場完就跑去吃午餐了，雖然有去比一些趣味競賽，也有領到全家禮券，但真的大家的熱情都不大，參加的人數都是硬湊出來的，活動的氛圍偏冷淡。我認為大一的時候還是可以參加一下，但感受過一次就好。

**Comment by XiangLuen** [comment]
大一的運動會，雖然好像學長都說大一的一定要參加，其實根本不用，我是覺得如果去的人太少會感覺這個系不重視校慶，但部分人其實都是來拿個早餐然後拿上系服就跑回宿舍躺。然後高年級後就知道這個活動可以不去，所以大二大三大四的基本就會直接回家或是去外面玩，免費的假為何不放。

**Comment by lichilun** [comment]
運動會記得學長姊都說要到，結果到現場很多人都沒有到，但是跟同系一起體驗運動會進場，凝聚了我們班級向心力，和同學一起比團體競賽更是刺激，也為我的大學生活記下美好的一筆。我覺得大學可以多嘗試，而且只要比賽有得名還可以多加操行成績（雖然躺在宿舍床睡覺，比較有吸引力）但我覺得可以去體驗一次看看！

**Comment by a414414** [comment]
其實運動會沒有硬性規定必須參加，如果能將這些時間拿去做更多事，像是跟朋友出去玩、讀書等等，也不錯，但我通常還是會參加運動會，因為運動會如果參加比賽有得名的話，可以拿到超商禮券，這可以節省很多開銷，很爽。

**Comment by Zheyuan987** [comment]
這個基本上是在浪費時間 不如去台北晃晃比較實在

**Comment by hollowha** [comment]
我只有去過3小時
